haskell/stack.yaml

```diff
@@ -11,7 +11,8 @@ extra-deps:
     https://gitlab.com/morley-framework/morley.git
     # ^ CI cannot use ssh, so we use http clone here
   commit:
-    48cbeec76b6126802d10e1d51da128982bae58e6 # master
+    # 48cbeec76b6126802d10e1d51da128982bae58e6 # master
+    08dab95815448dbe39942dbbff1a03fe9df113a7 # sras/#164-fa2-tests
```

Comment:
I think we won't merge those tests for now (before we fully sort things our in `stablecoin`). I think we agreed to do everything FA2-related in this repo and then move general things to `morley`.

Reply:
Yes. I had moved the tests here. But missed to update morley dep. Will do.